test/CrudTest.php

```diff
@@ -64,14 +64,13 @@ public function testUpdate()
 
     public function testIllegalAdd()
     {
-        $this->expectException(LdapException::class);
         $dn   = $this->createDn('ou=TestCreated,ou=Node2,');
         $data = [
             'ou'          => 'TestCreated',
             'objectClass' => 'organizationalUnit'
         ];
+        $this->expectException(LdapException::class);
         $this->getLDAP()->add($dn, $data);
-        $this->getLDAP()->delete($dn);
```

Comment:
Note: this was "bonus" code that ddoes not execute, because there is an exception that happens at the line above.

Reply:
Good catch! 👍